### PR TITLE
chore: [FX-2495] Remove critical vulnerabilities from picasso

### PR DESCRIPTION
**Pull request by LashaJini**
[FX-2495](https://toptal-core.atlassian.net/browse/FX-2495?atlOrigin=eyJpIjoiNDU1ODY3MWU3MjE1NDIxODk3Y2Q1YzEyNjI3MzJhNjYiLCJwIjoiaiJ9)

### Description

- Remove critical vulnerabilities from Picasso

### How to test

- Run `yarn audit --level critical` in Picasso repo and we should see 0 critical vulnerabilities:
Screenshot from 2022-02-17 19-22-19

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| Insert screenshots or screen recordings | Insert screenshots or screen recordings |

### Review

- [ ] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [ ] Annotate all `props` in component with documentation
- [ ] Create `examples` for component
- [ ] Ensure that deployed demo has expected results and good examples
- [ ] Ensure that tests pass by running `yarn test`
- [ ] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [ ] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

### CODE REVIEW

File: packages/picasso/src/FormControlLabel/styles.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ PicassoProvider.override(({ palette, typography }: Theme) => ({
       fontSize: '1em',
       color: palette.text.primary,
       lineHeight: '1.5em',
-      fontWeight: typography.fontWeightRegular,
+      fontWeight: typography.fontWeights.regular,
```
Comment (denieler) [inline packages/picasso/src/FormControlLabel/styles.ts:17]: what's the history of this change? 😃

Comment (LashaJini) [reply packages/picasso/src/FormControlLabel/styles.ts:17]: Typescript complained about `fontWeight` type when we used `fontWeightRegular`, also we use our own `typography.fontWeights.*` in other places.

Comment (denieler) [reply packages/picasso/src/FormControlLabel/styles.ts:17]: @LashaJini and does `typography.fontWeightRegular` returns the actual value? or `undefined`?

Comment (LashaJini) [reply packages/picasso/src/FormControlLabel/styles.ts:17]: It can be `undefined`.

Comment (LashaJini) [reply packages/picasso/src/FormControlLabel/styles.ts:17]: Screenshot from 2022-02-21 13-34-08

Comment (denieler) [reply packages/picasso/src/FormControlLabel/styles.ts:17]: it is undefined in Picasso? or has a value?

Comment (LashaJini) [reply packages/picasso/src/FormControlLabel/styles.ts:17]: In picasso, it's defined, and it has a type of `number`.

Comment (denieler) [reply packages/picasso/src/FormControlLabel/styles.ts:17]: and which value does it have? 😃

Comment (LashaJini) [reply packages/picasso/src/FormControlLabel/styles.ts:17]: 400

Comment (denieler) [reply packages/picasso/src/FormControlLabel/styles.ts:17]: then 👍  (I was thinking maybe it was broken before and we can remove this line)

**Review by denieler — APPROVED** [review]

**Review by TomasSlama — APPROVED** [review]

**Review by dmaklygin — APPROVED** [review]

File: .github/workflows/pr.yaml [hunk 2]
```diff
@@ -23,7 +23,7 @@ jobs:
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
     container:
-      image: cypress/included:8.7.0
+      image: cypress/included:9.5.0
```
Comment (LashaJini) [inline .github/workflows/pr.yaml:26]: We've upgraded cypress in dependencies.

**Pull request merged**